modules/lm-coursier/src/main/scala/lmcoursier/internal/ResolutionRun.scala

```diff
@@ -76,14 +83,14 @@ object ResolutionRun {
               dep
           }
         )
-        .withRepositories(params.repositories)
+        .withRepositories(repositories)
         .withResolutionParams(
           params
             .params
-            .addForceVersion(params.interProjectDependencies.map(_.moduleVersion): _*)
-            .withForceScalaVersion(isCompileConfig && params.autoScalaLibOpt.nonEmpty)
+            .addForceVersion((if (isCompileConfig) params.interProjectDependencies.map(_.moduleVersion) else Nil): _*)
```

Comment:
Doesn't it matter what the scoping of the `interProjectDependencies` are? You could have `dependsOn` scoped to `Test` for example.

Reply:
Test should be fine, that actually includes all the configs in the config sub-graph of compile (that is, runtime / test are included at least, scala-tool isn’t connected to compile, so isn’t included).

Reply:
We run one resolution per config sub-graph.

Reply:
Someone could also scope to some other sandbox configuration like `.dependsOn(something % "bar->bar")`

Reply:
Would you feel like submitting a fix for that? (checking which configurations are used in inter-project deps, and passing that around and using it around the code touched by this PR?

Reply:
I might come around to it eventually, but probably I won't be able to soon.